tekton/ci/interceptors/github/pkg/github/status.go

```diff
@@ -0,0 +1,31 @@
+package github
```

Comment:
This looks like it would be useful in pkg/interceptors

Reply:
Or as part of the interceptor sdk (cc @jmcshane)

Reply:
yeah, i think we can move everything from pkg/interceptors/sdk up to just pkg/interceptors. I think this means I agree with you that this function may make sense in tree as a status helper there